src/main/java/com/thealgorithms/dynamicprogramming/MinimumSumPartition.java

```diff
@@ -17,73 +19,31 @@
  */
 public class MinimumSumPartition {
```

Comment:
```suggestion
public final class MinimumSumPartition {
```

It is a _utility class_, so there are no reasons to _extend_ it.